app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list) -> int:
```

Comment:
Indicate a list of what instances you expect to obtain for each argument in the type annotation. Apply this change to other functions as well